solr/webapp/build.xml

```diff
@@ -19,12 +19,12 @@
   <description>Solr webapp</description>
 
   <property name="rat.additional-includes" value="**"/>
-  <property name="rat.additional-excludes" value="web/img/**"/>
+  <property name="rat.additional-excludes" value="web/img/**,*full*"/>
```

Comment:
That's a red herring - or a RAT herring.   RAT is a license checker.

Reply:
oh got. my initial exclude commit broke my web app locally. was trying to figure out what would work nothing had yet.